generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js

```diff
@@ -1,32 +1,37 @@
 import React from "react";
-
 import { NavBar } from "@components";
 
 import { NavBarProfile } from "./children";
-
-const navigation = [
-  {
-    id: "welcome",
-    icon: "img/icon/apps.svg",
-    label: "Welcome",
-    to: "/welcome"
-  },
-  {
-    id: "profile",
-    icon: "img/people.svg",
-    label: "Profile",
-    to: "/profile"
-  }
-];
+import { LanguageDropdown } from "@util-components";
 
 const AuthNavBar = props => {
+  const { t, onLanguageSelect } = props;
+  const navigation = [
+    {
+      id: "welcome",
+      icon: "img/icon/apps.svg",
+      label: t("navBar.welcome"),
+      to: "/welcome"
+    },
+    {
+      id: "profile",
+      icon: "img/people.svg",
+      label: "Profile",
```

Comment:
This label should be from i18n similar to Welcome

Reply:
done